build.gradle

```diff
@@ -485,7 +486,7 @@ project('test:system') {
     dependencies {
         // https://mvnrepository.com/artifact/com.mesosphere/marathon-client
         runtime group: 'ch.qos.logback', name: 'logback-classic', version: qosLogbackVersion
-        compile group: 'com.mesosphere', name: 'marathon-client', version: marathonClientVersion
+        compile group: 'com.mesosphere', name: 'marathon-client', version: marathonClientVersion, withoutSlf4jSimple
```

Comment:
Could you explain this change, please?

Reply:
The latest marathon-client v0.6.0 has a transitive dependency on group: 'org.slf4j', module: 'slf4j-simple'. We use logback as our slf4j implementation. Due to the transitive dependency we end up have both the implementations in the class path causing logging to not work.